### PR TITLE
Switch ProjectSearch PropTypes to Props

### PR DESCRIPTION
**Pull request by umohm1**
Associated Issue: #2804

### Summary of Changes

* Removed PropTypes to Props in ProjectSearch

### Test Plan

Tell us a little a bit about how you tested your patch.

Example test plan:

- [x]  yarn test
 - [x] yarn run flow

### CODE REVIEW

File: src/components/ProjectSearch/index.js [hunk 1]
```diff
@@ -4,8 +4,7 @@
 
 // @flow
 
-import PropTypes from "prop-types";
-import React, { Component } from "react";
+import React, { Component, PropTypes } from "react";
```
Comment (jasonLaster) [inline src/components/ProjectSearch/index.js]: do we need PropTypes?

Comment (bomsy) [reply src/components/ProjectSearch/index.js]: yep!

**Comment by jasonLaster** [comment]
Just pushed a simplified branch. Hope it helps :)

Sorry for the delay @umohm1

**Comment by jasonLaster** [comment]
Sorry for the delay. Turns out that we need to leave `contextTypes` in the code for react to be happy...

**Pull request merged**